tests/testthat/render-figs.R

```diff
@@ -39,16 +39,18 @@ asSvgFile <- function(funName, ggplot2Version = "2.2.1") {
 # Function to render all figures under different versions of ggplot2. Note, you
 # must have ggExtra version >= 0.6.1.9000 (commit 4b31c7cf or after) for these
 # figures to render correctly.
-renderAllFigsApply <- function(ggplot2Versions) {
+renderFigsApply <- function(ggplot2Versions) {
   withVersions(
     vdiffr = "0.1.1", fontquiver = "0.2.1", svglite = "1.2.0", code = {
       sapply(ggplot2Versions, function(ggplot2Version) {
         withVersions(ggplot2 = ggplot2Version, code = {
+          funList <- unlist(funList)
           sapply(
             names(funList), function(x) {
+              nm <- gsub(".*\\.", "", x)
```

Comment:
it's 3am and I can't think... what's this gsub doing?

Reply:
After unlisting `funList`, the function names appear in the form `<test name>.<function name>`. This line is stripping the test name component from these names. For example,  `"Misc issues are solved.theme bw"` becomes `"theme bw"`.